examples/Lorenz_Attractor.ipynb

```diff
@@ -36,8 +36,7 @@
     "init_notebook_mode(connected=True)\n",
     "\n",
     "# import data from openml\n",
-    "import openml\n",
-    "openml.config.start_using_configuration_for_example()"
+    "import openml"
```

Comment:
Are you sure the missing \n is correct?

Reply:
I removed the last line `"openml.config.start_using_configuration_for_example()"` and therefore the line `"import openml"` becomes the last one and does not require the `\n`.
Does it make sense?
Btw, the tests done on the Jupiter notebook are correctly passing: it seems to be correct.

Reply:
I removed the last line `"openml.config.start_using_configuration_for_example()"` and therefore the line `"import openml"` becomes the last one and does not require the `\n`.
Does it make sense?
Btw, the tests done on the Jupiter notebook are correctly passing: it seems to be correct.

Reply:
Perfect, sorry for not seeing it myself!